### DISCUSSION TITLE
2023 - 01- 30 오늘 할 일과 한일

### DISCUSSION THREAD

**Discussion by devjun63** [CodingGamePlayer/unstoppable#21] [General]
일정 리스트 페이지에 지역정보 추가
Gather Mapper 일정 리스트 현재시간보다 종료일이 빠른경우 제외시키기
일정 리스트 뷰 변경
메인페이지에만 calendar.js 사용
추가 작업은 댓글로 작성

**Comment by CodingGamePlayer** [comment]
👏
